apps/ledger-live-mobile/src/screens/Platform/v2/Catalog/LocalLiveApp.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { TouchableOpacity } from "react-native";
+import { Flex, ScrollContainer, Text } from "@ledgerhq/native-ui";
+import { AppIcon } from "../AppIcon";
+import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
+import { ScreenName } from "~/const";
+import { useNavigation } from "@react-navigation/core";
+import { NavigationProps } from "../types";
+
+export function LocalLiveApp({ localLiveApps }: { localLiveApps: LiveAppManifest[] }) {
+  const { t } = useTranslation();
+  const navigation = useNavigation<NavigationProps["navigation"]>();
+
+  return (
+    <>
+      <Flex flexDirection="row" alignItems="center" paddingX={4}>
+        <Flex flex={1} justifyContent="center">
+          <Text variant={"h4"} fontWeight={"semiBold"}>
+            {t("browseWeb3.catalog.section.locallyLoaded")}
+          </Text>
+        </Flex>
+      </Flex>
+
+      <ScrollContainer paddingLeft={6} horizontal showsHorizontalScrollIndicator={false}>
+        {localLiveApps.map(manifest => (
+          <TouchableOpacity
+            key={`${manifest.id}.${manifest.branch}`}
+            style={{
+              width: 70,
+              marginBottom: 16,
+              marginRight: 16,
+              justifyContent: "center",
```

Comment:
```suggestion
              marginTop: 16,
```

Reply:
There is already some space between the title and the app due to the title i think